### PR TITLE
Add docstrings to extensions

### PR DESCRIPTION
**Pull request by mike0sv**
Adding docs to extensions
For now chaining to #398 

close https://github.com/iterative/mlem/issues/412 as well

### CODE REVIEW

File: mlem/cli/apply.py [hunk 1]
```diff
@@ -36,38 +39,47 @@
...
+    False,
+    "-i",
+    "--import",
+    help="Try to import data on-the-fly",
+)
+option_import_type = Option(
+    None,
+    "--import-type",
+    "--it",
+    # TODO: change ImportHook to MlemObject to support ext machinery
```
Comment (aguschin) [inline mlem/cli/apply.py:59]: Create an issue?

**Review by aguschin — APPROVED** [review]
I assume you use this branch to generate docs in https://github.com/iterative/mlem.ai/pull/172 ?

File: mlem/cli/apply.py (hunk 1 above)
Comment (mike0sv) [reply mlem/cli/apply.py:59]: https://github.com/iterative/mlem/issues/252

**Comment by aguschin** [comment]
resolved merge conflict, but your tests fail @mike0sv

**Comment by mike0sv** [comment]
Will resolve and merge after #417

**Pull request merged**